lib/eye/notify/aws_sdk.rb

```diff
@@ -0,0 +1,40 @@
+require "aws-sdk-core"
+require "aws-sdk-core/ses"
+
+class Eye::Notify::AWS_SDK < Eye::Notify
```

Comment:
Use CamelCase for classes and modules.<br>Use nested module/class definitions instead of compact style.